charts/loki-distributed/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: loki-distributed
 description: Helm chart for Grafana Loki in microservices mode
 type: application
 appVersion: 2.2.0
-version: 0.30.0
+version: 1.0.0
```

Comment:
No need to use 1.0.0, moving to 0.31.0 is already considered a major release with helm charts.